tyk-headless/templates/deployment-pmp.yaml

```diff
@@ -47,14 +48,22 @@ spec:
         env:
           - name: REDIGOCLUSTER_SHARDCOUNT
             value: "{{ .Values.redis.shardCount }}"
+            {{- if .Values.mongo.enabled }}
```

Comment:
Wrap all of these in a single if statement since they all follow the same behaviour

Reply:
done